packages/lunarclient-app/lunarclient-app.pacscript

```diff
@@ -0,0 +1,57 @@
+name=lunarclient-app
+repology=("project: lunar-client")
+pkgver=3.1.3
+pkgrel=1
+epoch=1
+arch=("amd64")
+pkgdesc="PvP modpack for all modern versions of Minecraft"
+homepage=https://lunarclient.com
+license=(unknown)
+depends=(libfuse2)
+replace=("lunarclient" "lunar-client")
+breaks=("lunarclient-deb" "lunarclient-bin")
+maintainer="Mythbusters123 <atr10605@icloud.com>"
+_appimage="LunarClient-${pkgver}.AppImage"
+#_appimage="Lunar%20Client-${pkgver}.AppImage"
+# TEMPORARY ONLY UNTIL DOWNLOADING IS FIXED
+url="https://cdn.themyth.dev/LunarClient-3.1.3.AppImage"
+#url="https://launcherupdates.lunarclientcdn.com/${_appimage}"
+hash='f04f1364aa02f89c667f54e206326cc1f9bca8242ad2600a5cfbf748d91498b5'
+
+prepare() {
+	chmod +x "${_appimage}"
+	./"${_appimage}" --appimage-extract
+}
+
+build() {
+	# Adjust .desktop so it will work outside of AppImage container
+	sed -i -E \
+		"s|Exec=AppRun|Exec=env DESKTOPINTEGRATION=false /usr/bin/${pkgname}|" \
+		"squashfs-root/launcher.desktop"
+	# Fix permissions; .AppImage permissions are 700 for all directories
+	chmod -R a-x+rX squashfs-root/usr
+}
+
+package() {
+	# AppImage
+	sudo install -Dm755 \
+		"${srcdir}/${_appimage}" \
+		"${pkgdir}/opt/${pkgname}/${pkgname}.AppImage"
+
+	# Desktop file
+	sudo install -Dm644 \
+		"${srcdir}/squashfs-root/launcher.desktop" \
+		"${pkgdir}/usr/share/applications/${pkgname}.desktop"
+
+	# Icon images
+	sudo install -dm755 "${pkgdir}/usr/share/"
+	sudo cp -a \
+		"${srcdir}/squashfs-root/usr/share/icons" \
```

Comment:
```suggestion
		"${_appimage}" \
		"${pkgdir}/opt/${pkgname}/${pkgname}.AppImage"

	# Desktop file
	sudo install -Dm644 \
		"squashfs-root/launcher.desktop" \
		"${pkgdir}/usr/share/applications/${pkgname}.desktop"

	# Icon images
	sudo install -dm755 "${pkgdir}/usr/share/"
	sudo cp -a \
		"squashfs-root/usr/share/icons" \
```

Instead of `srcdir`, just use the current directory